english/coffee/drink-water/drink.md

```diff
@@ -3,4 +3,8 @@ but you have to keep drinking,
 now your stomach swells to the size of a hot air balloon. 
 You start feeling an urge to release some pressure...
 
-Do you find the nearest [bathroom](bathroom/bathroom.md)?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.